packages/preferences/src/components/preferences-modal-tabs/index.tsx

```diff
@@ -23,12 +23,21 @@ import { isRTL, __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { unlock } from '../../lock-unlock';
+/**
+ * External dependencies
+ */
+import type { ReactNode } from 'react';
 
 const { Tabs } = unlock( componentsPrivateApis );
 
 const PREFERENCES_MENU = 'preferences-menu';
 
-export default function PreferencesModalTabs( { sections } ) {
+export type PreferencesModalTabsProps = {
+	sections: { name: string; tabLabel: string; content: ReactNode }[];
```

Comment:
Let us get rid of that React import
```suggestion
	sections: { name: string; tabLabel: string; content: React.ReactNode }[];
```